docs/source/en/model_doc/madlad-400.md

```diff
@@ -56,6 +56,31 @@ One can directly use MADLAD-400 weights without finetuning the model:
 ['Eu amo pizza!']
 ```
 
+Note that in the input sentence, "`<2pt>`" is a token determining the target language of the translation. To see all supported language codes and their corresponding tokens:
```

Comment:
I would add this sentence before the code example so users know what it is beforehand

```suggestion
Include the target language token in the input sentence. `<2pt>` is the token for Portuguese.
```